pdns/dnssecinfra.cc

```diff
@@ -253,7 +253,9 @@ pair<unsigned int, unsigned int> DNSCryptoKeyEngine::testMakers(unsigned int alg
   else
     throw runtime_error("Can't guess key size for algorithm "+std::to_string(algo));
 
+  DTime dt; dt.set();
   dckeCreate->create(bits);
+  unsigned int udiffCreate = dt.udiff() / 100;
```

Comment:
We only create once so unless I'm mistaken the `/ 100` part should go.

Reply:
blind copy pasting - maybe we should create 100 times too?

Reply:
That might make sense, yes. Verify as well, I guess?

Reply:
amended